src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -163,8 +163,6 @@ def test_complex_code():
     sched = psy.invokes.invoke_list[0].schedule
     loops = sched.walk(sched.children, nemo.NemoLoop)
     assert len(loops) == 5
-    cblocks = sched.walk(sched.children, CodeBlock)
```

Comment:
pylint complains that `CodeBlock` is now unused. Would you mind fixing the other things it complains about while you're here even though they're not your fault?

Reply:
Fixed all but: `Unable to import  ***` and ` Access to a protected member ***` warnings.